packages/block-editor/src/components/inspector-controls/index.native.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ifBlockEditSelected } from '../block-edit/context';
+import { BlockSettingsButton } from '../block-settings';
+
+const { Fill, Slot } = createSlotFill( 'InspectorControls' );
+
+const FillWithSettingsButton = ( { children, ...props } ) => {
+	return (
+		<>
+			<Fill { ...props }>{ children }</Fill>
+			{ children && ( <BlockSettingsButton /> ) }
```

Comment:
I think it would be safer to check `React.Children.count( children ) > 0` rather than see if it's truthy. I can't think of many examples, but if I understand correctly, `children` could be a function (truthy) that returned no children

Reply:
you're right will change that

Reply:
done